packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -389,14 +621,30 @@ String _calculateSha(File file) {
   return sha;
 }
 
+void _printUndefinedTask(GradleProject project, BuildInfo buildInfo) {
```

Comment:
According to https://codecov.io/gh/flutter/flutter/pull/35217/diff#D21-624, this has no test coverage